src/services/penaltyDocuments.js

```diff
@@ -44,14 +44,20 @@ export default class PenaltyDocument {
 		const dbGet = this.db.get(params).promise();
 
 		dbGet.then((data) => {
-			if (!data.Item) {
-				callback(null, createResponse(404, 'ITEM NOT FOUND'));
+			console.log(JSON.stringify(data.Item, null, 2));
+			console.log('before object check');
+			if (!data.Item || this.isEmpty(data.Item)) {
+				console.log('404 error here we come');
```

Comment:
@rossfauldsbjss  Not sure if you meant to leave this console.log, I like the funny message!

Reply:
will clean up console logs.